src/js/pages/services/DeploymentsTab.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+import AlertPanel from '../../components/AlertPanel';
+
+var DeploymentsTab = React.createClass({
```

Comment:
We try to use ES6 whenever we have a choice :) What documentation are you referring too?